src/main/java/nl/tudelft/ewi/devhub/server/database/entities/Delivery.java

```diff
@@ -44,6 +44,14 @@
         public String getTranslationKey() {
             return translationKey;
         }
+        
+        /**
+         * Get the lowercase name for freemarker.
+         * @return {@link Enum#name() name()}{@link String#toLowerCase() .toLowerCase()}
+         */
+        public String lowerCaseName() {
```

Comment:
I found out FreeMarker didn't parse chaining of methods, so I created this to make it look more natural than ALL-CAPS.